crates/wash-lib/src/common.rs

```diff
@@ -72,12 +133,68 @@ pub async fn find_actor_id(
         Err(FindIdError::MultipleMatches(
             all_matches
                 .into_iter()
-                .map(|(id, friendly_name)| {
-                    if let Some(name) = friendly_name {
-                        format!("{} ({})", id, name)
-                    } else {
-                        id.into_string()
-                    }
+                .map(|(id, friendly_name)| Match {
+                    id: id.to_string(),
+                    friendly_name,
+                })
+                .collect(),
+        ))
+    } else {
+        // SAFETY: We know we have exactly one match at this point
```

Comment:
Same nit about that `[..]` pattern here, it would probably make this bit easier to read/remove requirement of the written assurance (even though it is legitimately impossible to not be the case there)

Reply:
See previous comment

Reply:
Resolving, this isn't blocking